m2e-maven-runtime/org.eclipse.m2e.maven.runtime/pom.xml

```diff
@@ -163,6 +163,7 @@
 								META-INF.plexus;-noimport:=true;x-internal:=true,\
 								META-INF.sisu;-noimport:=true;x-internal:=true,\
 								org.apache.maven.*;provider=m2e;mandatory:=provider,\
+								org.apache.commons.cli;provider=m2e;mandatory:=provider,\
```

Comment:
does maven uses common.cli internally? I can't find a reference in your code about it.

Reply:
It used to be used in `MavenConfigParser`. But I moved the 'Parser' to the m2e.maven.runtime. This way it is possible to access package private methods from MavenCLI and to reuse more of its code.
Then this package does not have to be exported as well.